src/all/batoto/src/eu/kanade/tachiyomi/extension/all/batoto/BatoTo.kt

```diff
@@ -392,7 +393,14 @@ open class BatoTo(
     }
 
     private fun checkChapterLists(document: Document): Boolean {
-        return document.select(".episode-list > .alert-warning").text().contains("This comic has been marked as deleted and the chapter list is not available.")
+        val chapterListElements = document.select(chapterListSelector())
+        if (chapterListElements.isEmpty()) {
+            val alertWarningElement = document.select(".episode-list > .alert-warning").firstOrNull()
+            if (alertWarningElement != null && alertWarningElement.text().contains("This comic has been marked as deleted and the chapter list is not available.")) {
+                throw IOException("This comic was deleted.")
+            }
+        }
+        return false
```

Comment:
this check should be in the parse method, otherwise the attempt to fetch chapters from alt source won't work

Reply:
ngl i tried to figure this out for 4h straight and nothing went right
will try again tomorrow